tests/integration/models/application.spec.coffee

```diff
@@ -581,3 +581,85 @@ describe 'Application Model', ->
 					.then =>
 						promise = balena.models.application.isTrackingLatestRelease(@application.id)
 						m.chai.expect(promise).to.eventually.be.true
+
+	describe 'given a multicontainer application with a single offline device', ->
+
+		givenMulticontainerApplication()
+
+		describe 'balena.models.application.getWithDeviceServiceDetails()', ->
+
+			it 'should retrieve application and it\'s devices along with service details', ->
+				balena.models.application.getWithDeviceServiceDetails(@application.id).then (applicationDetails) =>
```

Comment:
Let's also test a bit the application details.
Something like this should work:
```
m.expect(_.omit(applicationDetails, 'owns__device')).to.become(_.omit(@application, 'owns__device'))
```